scalameta/parsers/js/src/test/scala/scala/meta/parsers/JSFacadeSuite.scala

```diff
@@ -193,5 +195,56 @@ class JSFacadeSuite extends FunSuite {
     check(parsed, expected)
   }
 
+  test("default dialect is Scala 2.11") {
+    val code =
+      """|List(
+         |  1,
+         |  2,
+         |)""".stripMargin
+    val parsedDefaultDialect = JSFacade.parseStat(code)
+    val expected = d(
+      "error" -> "illegal start of simple expression",
+      "pos" -> pos(16, 17)
+    ).asInstanceOf[js.Dictionary[Any]]
+    check(parsedDefaultDialect, expected)
+  }
+
+  test("inexisting dialects fallback to Scala 2.11") {
+    val code =
+      """|List(
+         |  1,
+         |  2,
+         |)""".stripMargin
+    val parsedDefaultDialect = JSFacade.parseStat(code, js.Dictionary("dialect" -> "wrong"))
```

Comment:
Right, probably if you pass a wrong one you want to know. It makes sense I'll report an error (instead of throwing an exception)